apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js

```diff
@@ -136,6 +136,18 @@ export default class CodeReviewDataApi {
     });
   }
 
+  deleteCodeReviewComment(commentId) {
```

Comment:
This can also just be 
```
return $.ajax({
        url: `/code_review_notes/${commentId}`,
        type: 'DELETE',
        headers: {'X-CSRF-Token': this.token}
      })
```
Since it returns a promise